servicemesh-install/topics/tutorial-grafana.adoc

```diff
@@ -1,10 +1,7 @@
 [[grafana-tutorial]]
 = Grafana tutorial
 
-////
-TODO
-Need a short overview of Grafana.
-////
+Grafana is an open source a tool for creating monitoring and metric analytics & dashboards.  Grafana lets you to query, visualize, alert on  your metrics no matter where they are stored: Graphite, Elasticsearch, OpenTSDB, Prometheus, or InfluxDB.  With Istio you get monitoring via Prometheus and Grafana out of the box.
```

Comment:
s/Grafana lets you to/You use Grafana to
s/With Istio you get monitoring via Prometheus and Grafana out of the box./Istio includes monitoring via Prometheus and Grafana.  (IBM Style says not to use out of the box.)